src/Magnum/Platform/WindowlessGlxApplication.cpp

```diff
@@ -94,7 +103,7 @@ WindowlessGlxContext::WindowlessGlxContext(const WindowlessGlxContext::Configura
         #endif
         0
     };
-    _context = glXCreateContextAttribsARB(_display, configs[0], nullptr, True, contextAttributes);
+    _context = glXCreateContextAttribsARB(_display, configs[0], ctx, True, contextAttributes);
```

Comment:
i think just 

```suggestion
    _context = glXCreateContextAttribsARB(_display, configs[0], configuration.sharedContext(), True, contextAttributes);
```

everywhere is descriptive enough, no need for the explanatory comment.